cmd/rukpakctl/cmd/bundle.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	"github.com/operator-framework/rukpak/cmd/rukpakctl/utils"
+)
+
+type BundleOptions struct {
+	k8sclient  *kubernetes.Clientset
+	k8sdynamic dynamic.Interface
+}
+
+var bundleOpt BundleOptions
+
+// bundleCmd represents the bundle command
+var bundleCmd = &cobra.Command{
+	Use:   "bundle <bundle name>, <manifest directory>",
+	Short: "bundle command creates rukpak bundle resource.",
+	Long:  `bundle command creates rukpak bundle resource with specified contents.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		err := bundle(bundleOpt, args)
+		if err != nil {
+			fmt.Printf("bundle command failed: %+v\n", err)
+		}
+	},
+}
+
+func init() {
+	createCmd.AddCommand(bundleCmd)
+	homeDir := os.Getenv("HOME")
+	config, err := clientcmd.BuildConfigFromFlags("", homeDir+"/.kube/config")
```

Comment:
I'm not sure this is the best way to do this. I believe kubectl has an exported function to get config from the machine, could we use that instead? 

If we do rely on a `HOME` env variable then we would want to document it

Reply:
Thanks for review.  I'll look into it.

Reply:
I fixed it.